protobuf/syft_proto/types/torch/v1/tensor_data.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package syft_proto.types.torch.v1;
+
+import "syft_proto/types/syft/v1/id.proto";
+import "syft_proto/types/syft/v1/shape.proto";
+
+message TensorData {
+    syft_proto.types.syft.v1.Id id = 1;
+    syft_proto.types.syft.v1.Shape shape = 2;
+    
+    string dtype = 3;
```

Comment:
How about making tensor type enum here?
This way we could clearly state types supported by protocol.

Reply:
I considered that, but that makes it somewhat harder to [set the correct field](https://github.com/OpenMined/PySyft/pull/2869/files#diff-370839d90a953efe68abc2dde509e961R106) on a `TensorData` message. I do like the idea of locking down what dtypes can be set there though.